src/tree.ts

```diff
@@ -206,19 +184,24 @@ const makeSrcSet = (
     return createFileLink(handle, transformOptions, options.useValidator)();
   }
 
-  const resolutions: any[] = R.map(R.ifElse(
-    R.is(Number),
-    (n: number) => `${n}w`,
-    R.identity,
-  ), options.resolutions);
+  const resolutions = options.resolutions.map((val: any) => {
+    const resString = typeof val === 'number' ? `${val}w` : val;
```

Comment:
You dont need to declare variable here, just return value, You can do one liner as well from this method